topics/climate/tutorials/climate-101/tutorial.md

```diff
@@ -0,0 +1,399 @@
+
+At the moment, there are [54 ECVs](https://gcos.wmo.int/en/essential-climate-variables).
+
+*Source: [https://gcos.wmo.int/en/essential-climate-variables](https://gcos.wmo.int/en/essential-climate-variables)*
+
+> ### {% icon hands_on %} Hands-on: Essential Climate Variables
+>
+>    1. **Copernicus Essential Climate Variables** {% icon tool %} with the following parameters:
+>        - *"Variable(s)"*: precipitation
+>        - *"Select type of data"*: Anomaly
+>        - *"Select year(s)"*: `1980` and `2018`
+>        - *"Time aggregation"*: yearly
+>        - *"Select month"*: `July`
+>        Rename the resulting file to `precipitation_anomalies_july_1980_2018.nc`
+>    2. **map plot gridded (lat/lon) netCDF data** {% icon tool %} with the following parameters:
+>        - *"input with geographical coordinates (netCDF format)"*: `precipitation_anomalies_july_1980_2018.nc`
+>        - *"variable name as given in the netCDF file": `tp`
+>        - And finally in `Advanced Options` change:
+>            - *"multiple times"*: `Yes`
+>            - *"comma separated list of indexes for fields to plot"*: 0,1
+>            - *"number of rows for subplot grid"*: 2
+>            - *"subplot title (repeated on each subplot)"*: precipitation anomalies (m/day)
+>            - *"colormap"*: PiYG
+>    > ### {% icon question %} Precipitation anomalies
+>    > 
+>    > 1. Do you observe any significant precipitation anomalies in 1979 versus 2018?
+>    > 2. Do we have sufficient information to make any conclusions on the change in climate?
+>    > 
+>    > > ### {% icon solution %} Solution
+>    > >
+>    > >
+>    > {: .solution}
+>    {: .question}
+>
+{: .hands_on}
+
+
+## Past, present and future climate?
+
+
+When we talk about climate data, the type of data can vary significantly. We have very little actual observations at the scale of climate and usually not covering a large area. In addition to observations, we can make use of:
+- Re-analyses where observations and numerical modelling are combined together. 
+- Climate models.
+
+Observations and re-analyses provide information about the past and current climate while climate models can provide past, current and future climate information.
+When it comes to future climate, we usually need to make some assumptions (such as how much CO2 emissions, etc.) and make different scenarios e.g. we run climate models using different assumptions and look at future trends under each of these scenarios: this is what we call **climate projections**. Climate projections will be discussed in a separate Galaxy tutorial.
```

Comment:
"and make different scenarios" sounds strange to me ... simulate different scenarios maybe?